.gitignore

```diff
@@ -49,3 +49,6 @@ npm-debug.log
 
 # yarn
 yarn.lock
+
+# pnpm
+shrinkwrap.yaml
```

Comment:
Please do not include this change to the `.gitignore`. Thanks!

Reply:
Why shouldn't I?

Reply:
Please send non-types (i.e. infrastructure) changes in a separate PR along with reasoning for why the change is needed. Mixing/matching different changes makes it harder to smartly roll back changes if needed.

Reply:
OK, I just moved it to https://github.com/DefinitelyTyped/DefinitelyTyped/pull/26192